languages/HoldingStatus/de.ini

```diff
@@ -2,3 +2,14 @@ service_available_presentation = "Nur Präsenznutzung"
 service_loan = "Ausleihexemplar"
 service_presentation = "Präsenzexemplar"
 services_available_html = "Verfügbar als %%list%%"
+status_1	= "Nicht ausleihbar"
```

Comment:
If you want to set an unavailable status for a large number of values, would it be cleaner to map this inside the ILS driver based on a configuration setting in the driver .ini file rather than creating the need to potentially maintain many parallel values in many parallel language files?